.github/workflows/build.yaml

```diff
@@ -57,21 +57,20 @@ jobs:
       packages: write
     steps:
       - name: Checkout (GitHub)
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
 
       - name: Login to GitHub Container Registry
-        uses: docker/login-action@v2
+        uses: docker/login-action@v3
         with:
           registry: ghcr.io
           username: ${{ github.repository_owner }}
-          password: ${{ secrets.GITHUB_TOKEN }}
+          password: ${{ github.token }}
 
       - name:
         uses: devcontainers/ci@v0.3
         with:
           imageName: ghcr.io/mt-caret/polars-ocaml
           cacheFrom: ghcr.io/mt-caret/polars-ocaml
-          push: always
```

Comment:
It's enough to push from just a single job, so you can get rid of this.

Reply:
Ah sweet, thanks!